packages/components/src/components/alert/test/html.mock.ts

```diff
@@ -14,6 +14,26 @@ export const getAlertHtml = (props: AlertProps, innerHTML = '', additionalHTML =
 	const type: string = props._type !== undefined ? props._type : 'default';
 	props._type = props._type || 'default';
 	props._variant = props._variant || 'msg';
+
+	let kolIcon = '';
```

Comment:
Das sind keine Icons, das sind die Texte für das ariaLabel, welche die `translate`-Funktion ausspuckt.
In der Testumgebung gibt es (gewollt) keine echten Übersetzungen, daher gibt `translate` einfach den Key zurück.

Reply:
Ich fand das Refactoring eigentlich gut, mir ging um nur um die Benennung